packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,158 @@
+| Done                 | Response                        |
+
+
+## Common parts of the flow
+
+Some parts of the sign-in flow are the same:
+
+- Add the SDK to your app.
+- Configure the SDK.
+- Initialize the client and start the flow.
+- Process the response.
+- Request a token.
+- Sign the user out.
+
+### Add the SDK
+
+<StackSnippet snippet="adddependency" />
+
+### Create and manage configurations
+
+The values for a configuration object are:
+
+| Value         | Description |
+| :------------ | :---------- |
+| Issuer        | The Oauth 2.0 URL for the Okta org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin Console.  |
+| Redirect URI  | A callback URI for launching the mobile app, such as `com.example.oie-123456:/callback`. |
+| Scopes        | A space separated list of the requested access scopes for the app. For more information, see [Scopes in OpenID Connect & OAuth 2.0 API](https://developer.okta.com/docs/reference/api/oidc/#scopes).|
+
+The configuration information in a shipping app is usually static. You can initialize the configuration values directly in the code or by reading them in from a file. During development you may want to provide a way to edit configuration values.
+
+<StackSnippet snippet="loadingaconfiguration" />
+
+### Start sign-in
+
+Start the sign-in flow by creating an SDK client using a configuration, and then requesting the first step.
+
+<StackSnippet snippet="initializingsdksession" />
+
+### Process the response
+
+The steps for processing a response are:
+
+1. Check for a successful sign-in flow.
+1. Check for messages, such as an invalid password.
+1. Check for remediations.
+1. Process the remediations.
+1. Process the authenticators.
+
+After the user enters the required information, update the remediation and request the next step.
+
+<StackSnippet snippet="processresponse" />
+
+### Complete the sign in flow
+
+Check the response for a successful sign in at each step of the flow. When the user is signed in, exchange the remediation for an access token, and then exit the flow.
+
+<StackSnippet snippet="gettingatoken" />
+
+### Sign out a user
+
+Sign out a user by revoking the access token granted when the user signed in, and by revoking a refresh token if one exists.
+
+<StackSnippet snippet="signingout" />
+
+## Sign-in flow UI
+
+There are two general approaches for adding the sign-in flow to your app: add a fixed number of authentication methods, possibly in a fixed order, or generate the user interface dynamically based on the response at each step.
+
+For a fixed number of methods (a static UI), you create appropriate views and populate them from the response. Using a static UI introduces risk as both the authentication methods and the order may be changed at any time by an administrator of your Okta org. There are three requirements to reduce that risk:
```

Comment:
The suggestion changes the semantics of the paragraph. The first suggestion changes the subject of the first paragraph which is the static UI, not creating the appropriate views. The second suggestion reduces the urgency of the story of this article: create dynamic views because anything else has lots of risks including that the admin can change things at anytime by someone else.

Unless there's reasons to the contrary I'll keep the original.

Reply:
Agreed.